packages/quantic/force-app/examples/main/lwc/exampleQuanticRefineToggle/exampleQuanticRefineToggle.js

```diff
@@ -22,6 +22,20 @@ export default class ExampleQuanticRefineToggle extends LightningElement {
         'Indicates whether the refine modal should be opened in full screen.',
       defaultValue: false,
     },
+    {
+      attribute: 'withoutFacets',
```

Comment:
:thinking: The options declared here were meant to match the component under test options, not to be used as testing utilities.

I think it would be cleaner to use an action button for the "without facets" test. I could be a "Perform search with no facets" button. This way, it would be clear to a user that the "without facets" is the result of the performed query, and not an option you can set on the component to change its behavior.

The "facets without inputs" case is a little more tricky to handle because the option is meant to configure the example component, not the component under test. I guess you can keep this one like this for now, but we should improve the example component framework to support this. We'd need the same feature to easily switch the use case under test (i.e., search, insight, case assist, etc)

Reply:
`WithoutFacets` and `FacetsWithoutInputs` removed from the options and are implemented as action buttons:
https://user-images.githubusercontent.com/86681870/187200252-bf114320-ad48-4749-a1e5-16bcf43bb747.mov